types/chrome/index.d.ts

```diff
@@ -1898,6 +1921,11 @@ declare namespace chrome.cookies {
 
     export interface Details {
```

Comment:
You don't need to do this, but the docs call this interface `CookieDetails`. Would you mind changing that name and throwing some docs on it?

Reply:
This type is exported. Would not it be a breaking change if I rename it? How do you approach breaking changes in DT?

Reply:
I added the docs. Will be happy to rename to `CookieDetails`, too, once you confirm it's ok to include a potentially breaking change in this PR

Reply:
Unfortunately, chrome types are a bit of a weirder thing in DT. Normally, `@types` follows the package versioning (generally semver) until patches (which are incremented on each release of the types package). The thing with chrome is that there is no real versioning on chrome API's. You could argue that Chrome versions are version enough to follow them, but you need to remember that extensions have to run across multiple major versions of chrome. Also, you need to consider that chrome extensions can run on all chromium based browsers (Chrome, Edge, Brave, ect.). Our solution is to forgo versioning, and just call everything 0.0.x (I think we're closing in on 300 now). It's annoying but essentially breaking changes happen all the time when chrome API's are updated. Chrome is good with giving backwards compatibility and letting developers update their code as API's come into/out of existence, so we have a bit of built in backwards compatibility. In this case, we made a mistake naming something (or the name was changed), so we should 100% fix that, and there's no precedent for versioning breaking changes in this world, so you're all good to go.

TL;DR: Yes, it's fine to include a breaking change as long as it follows the chrome API's.

Reply:
Got it, thanks for the context!

It's done, too

Reply:
Hello @nixolas377 just a quick question ☺️

Since we're not tied to an API version, would it be possible to bump the package version to 0.1.999 to indicate a breaking change ?

Reply:
I wouldn't see a reason why, to me I think the reason we do this versioning is to get rid of any confusion with a version of Chrome. These types are for the latest version of chrome (as discussed above they have backwards compatibility built in), and since there's so many chromium browsers with the same/extremely similar apis, all with competing versioning, we don't want to confuse anyone.